core/integration/bookkeeping/api.go

```diff
@@ -66,7 +66,7 @@ func Instance() *BookkeepingWrapper {
 	return instance
 }
 
-func (bk *BookkeepingWrapper) CreateRun(activityId string, nDetectors int, nEpns int, nFlps int, runNumber int32, runType string, timeO2Start time.Time, timeTrgStart time.Time, dd_flp bool, dcs bool, epn bool, epnTopology string, detectors string) error {
+func (bk *BookkeepingWrapper) CreateRun(activityId string, nDetectors int, nEpns int, nFlps int, runNumber int32, runType string, dd_flp bool, dcs bool, epn bool, epnTopology string, detectors string) error {
```

Comment:
Pedantic comment: Is there a reason for snake case on `dd_flp`? If not I'd use camel case to keep it consistent

Reply:
No, there isn't. You're right, I'll change this!